packages/cli-server-api/src/index.ts

```diff
@@ -79,6 +79,7 @@ export function createDevServerMiddleware(options: MiddlewareOptions) {
         '/message',
       );
       broadcast = messageSocket.broadcast;
+      isDebuggerConnected = debuggerProxy.isDebuggerConnected;
```

Comment:
cc @fson the debugger connected state was not updated, likely an oversight when moving code around